server/api/rounds.py

```diff
@@ -241,17 +241,17 @@ def sampled_batch_results(election: Election,) -> BatchTallies:
     }
 
 
-def batches_times_sampled(election: Election) -> Dict[Tuple[str, str], int]:
-    sampled_batch_draw_counts = (
+def sampled_batches_by_ticket_number(election: Election) -> Dict[str, Tuple[str, str]]:
+    batches_by_ticket_number = (
         SampledBatchDraw.query.join(Batch)
         .join(Jurisdiction)
         .filter_by(election_id=election.id)
-        .group_by(Jurisdiction.name, Batch.name)
-        .values(Jurisdiction.name, Batch.name, func.count())
+        .order_by(SampledBatchDraw.ticket_number)
+        .values(SampledBatchDraw.ticket_number, Jurisdiction.name, Batch.name)
```

Comment:
This is a really good catch, I hadn't considered that we weren't using sampling order.